frontend/test/metabase/scenarios/admin/databases/list.cy.spec.js

```diff
@@ -131,4 +131,29 @@ describe("scenarios > admin > databases > list", () => {
 
     cy.findByRole("status").should("not.exist");
   });
+
+  it.skip("should handle malformed (null) database details (metabase#25715)", () => {
+    cy.request("GET", "/api/database/1").then(({ body }) => {
+      const stubbedResponse = {
+        ...body,
+        details: null,
```

Comment:
The page completely blows up because the expected type on FE is an object.
If I stub `details` as an empty object (simply to satisfy the schema), the page renders with the empty connection string field.
![image](https://user-images.githubusercontent.com/31325167/217393690-63d7648a-fe85-41c6-a1eb-ef4e8184b309.png)